jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MultiPart.java

```diff
@@ -1370,6 +1365,8 @@ private boolean parseContent(Content.Chunk chunk)
             int boundaryOffset = boundaryFinder.match(buffer);
             if (boundaryOffset >= 0)
             {
+                // Output as content the previous partial match, if any.
+                notifyCRContent();
```

Comment:
I don't think this is correct.

If we matched `\n--Boundary` at position 0. Then the previous `\r` is part of the boundary and should not be emitted as content.

Reply:
Good catch, fixed.